ganga/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -142,6 +142,7 @@ class GaudiExec(IPrepareApp):
         'jobScriptArchive': GangaFileItem(defvalue=None, hidden=1, copyable=0, doc='This file stores the uploaded scripts which are generated fron this app to run on the WN'),
         'useGaudiRun':  SimpleItem(defvalue=True, doc='Should \'options\' be run as "python options.py data.py" rather than "gaudirun.py options.py data.py"'),
         'platform' :    SimpleItem(defvalue='x86_64-slc6-gcc62-opt', typelist=[str], doc='Platform the application was built for'),
+        'autoDBtags' :  SimpleItem(defvalue=False, doc='Automatically set database tags for MC'),
```

Comment:
I suggest to change default to `True`. Otherwise the feature, most beneficial for new users, will never be used.

Reply:
Perhaps - although this can change the result of the job so I would rather have the user make a conscious decision to use it.